locations/jclouds/src/test/java/org/apache/brooklyn/location/jclouds/provider/AwsEc2LocationLiveTest.java

```diff
@@ -28,6 +30,8 @@
  */
 public class AwsEc2LocationLiveTest extends AbstractJcloudsLocationTest {
 
+    private static final Logger log = LoggerFactory.getLogger(AwsEc2LocationLiveTest.class);
```

Comment:
The logger is not used here. Can be removed

Reply:
also used in test